make/golang-v1.mk

```diff
@@ -150,6 +150,26 @@ else \
 fi;
 endef
 
+# golang-bump-major-version bumps the module's major version in go.mod, 
+# VERSION, swagger.yml, and non-vendored Go files.
+define golang-bump-major-version
+# bump go.mod
+$(eval $@OLD_VERSION := $(shell cat swagger.yml | grep 'version:\s[0-9].[0-9].[0-9]' | sed 's/[:|[:alpha:]|[:space:]]//g' | cut -d. -f1 ||\ 
+cat VERSION.yml | sed 's/[:|[:alpha:]|[:space:]]//g' | cut -d. -f1))
+$(eval $@NEW_VERSION := $(shell echo $$(($($@OLD_VERSION)+1))))
+sed -i '' '1s/v$(1)/v$($@NEW_VERSION)/' go.mod 
```

Comment:
what's the risk this hits unrelated things?

Reply:
my understanding is that it should only replace the first line because the sed format uses `1s/` & the first line of the go.mod file will always be the repos module name